scripts/prep_debug_env.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+DEBUG_PORT=$1
+PROJECT=${ASB_PROJECT:="ansible-service-broker"}
+
+oc rollout pause dc/asb -n ${PROJECT}
```

Comment:
The broker's deployment config, when deployed via the broker-apb, will not be named `asb`. In https://github.com/openshift/ansible-service-broker/pull/939 every script that I could find that will deploy the broker will use the apb and https://github.com/fusor/catasb/pull/242 changes catasb to enable the broker (which also uses an apb).